pkg/handler/rpms.go

```diff
@@ -198,3 +200,46 @@ func (rh *RpmHandler) detectRpmsPresence(c echo.Context) error {
 
 	return c.JSON(200, apiResponse)
 }
+
+// listSnapshotErrata godoc
+// @Summary      List Snapshot Errata
+// @ID           listSnapshotErrata
+// @Description  List errata in a repository snapshot.
+// @Tags         snapshots
+// @Accept       json
+// @Produce      json
+// @Param		 uuid	path string true "Snapshot ID."
```

Comment:
format looks strange, is there a tab in "uuid path" ?

Reply:
fixed